alfajores.env

```diff
@@ -42,6 +42,12 @@ IS_CUSTOM_CHAIN=true
 # The path can be absolute, or relative to the docker-compose.yml file.
 DATADIR_PATH=./envs/${NETWORK_NAME}/datadir
 
+# Specifies the endpoint of the eigenda proxy to use. If this is unset then a local eigenda proxy will be used.
+EIGENDA_PROXY_ENDPOINT=https://eigenda-proxy.alfajores.celo-testnet.org
+
+# Set to 0 if EIGENDA_PROXY_ENDPOINT is defined otherwise epmpty. This shouldn't be set manually.
+USE_LOCAL_EIGENDA_PROXY=${EIGENDA_PROXY_ENDPOINT:+0}
```

Comment:
From the var naming, it is not expected that a local proxy is used if `USE_LOCAL_EIGENDA_PROXY` is unset. Maybe call it `USE_LOCAL_EIGENDA_PROXY_IF_UNSET`?